src/orchard/namespace.clj

```diff
@@ -104,8 +125,16 @@
 ;; These methods search sources on the classpath. Non-classpath source
 ;; files, documentation code, etc within the project directory are ignored.
 
+(defn jvm-clojure-resource-name->ns-form
```

Comment:
I'm not sure that `jvm` here is needed, given how much of the codebase in Orchard is (Java) Clojure-specific anyways.

Reply:
I'd hope that to be decreasingly the case over time!

Reply:
Yeah, but as noted elsewhere I think it'd be weird to prefix all platform-specific functions here. Probably it'd be better to have implementation specific namespaces and some platform-independent ns that wraps around those.

Reply:
Note that there was already an existing `jvm-clojure-resource-name->ns-name` defn, so it's consistent.

Similarly, I renamed `project-jvm-ns-forms` -> `project-ns-forms` for consistency with what's here